packages/image_picker/image_picker/README.md

```diff
@@ -11,6 +11,9 @@ First, add `image_picker` as a [dependency in your pubspec.yaml file](https://fl
 
 ### iOS
 
+Starting with version **0.8.1** the iOS implementation uses PHPicker to pick (multiple) images on iOS 14 or higher.
+As a result of implementing PHPicker it becomes impossible to pick HEIC images on the iOS simulator in iOS 14+. This is a known issue. Please test this on a real device, or test with non-HEIC images until Apple solves this issue.[63426347 - Apple known issue](https://www.google.com/search?q=63426347+apple&sxsrf=ALeKk01YnTMid5S0PYvhL8GbgXJ40ZS[…]t=gws-wiz&ved=0ahUKEwjKh8XH_5HwAhWL_rsIHUmHDN8Q4dUDCA8&uact=5) 
```

Comment:
This is a Google Search URL, is there a direct link to the Apple issue tracker? I can only find forum threads like this: https://developer.apple.com/forums/thread/658135

Reply:
No, there is not an issue tracker. Not one publicly available. It seems those issues can contain private data which is why they are private.

Reply:
Yes, I put the Google Search URL, because I could not find the direct link for the 63426347 issue.

Reply:
Is https://developer.apple.com/forums/thread/658135 a better link? @danielroek